examples/simple_intersection/example_intersection.cpp

```diff
@@ -15,153 +15,61 @@
+
+  Kokkos::View<ArborX::Box *, MemorySpace> boxes("Example::boxes", 4);
+  auto boxes_host = Kokkos::create_mirror_view(boxes);
+  boxes_host[0] = {{0, 0, 0}, {1, 1, 1}};
+  boxes_host[1] = {{1, 0, 0}, {2, 1, 1}};
+  boxes_host[2] = {{0, 1, 0}, {1, 2, 1}};
+  boxes_host[3] = {{1, 1, 0}, {2, 2, 1}};
+  Kokkos::deep_copy(boxes, boxes_host);
+
+  // -----------
+  // |    | 1  |
+  // |    |  0 |
+  // |----2----|
+  // |    |    |
+  // |    |    |
+  // -----------
+  Kokkos::View<decltype(ArborX::intersects(ArborX::Point())) *, MemorySpace>
+      queries("Example::queries", 3);
+  auto queries_host = Kokkos::create_mirror_view(queries);
+  queries_host[0] = ArborX::intersects(ArborX::Point{1.8, 1.5, 0.5});
+  queries_host[1] = ArborX::intersects(ArborX::Point{1.3, 1.7, 0.5});
+  queries_host[2] = ArborX::intersects(ArborX::Point{1, 1, 0.5});
+  Kokkos::deep_copy(queries, queries_host);
+
+  ExecutionSpace space;
+
+  ArborX::BVH<MemorySpace> const tree(space, boxes);
 
-  std::cout << "Create grid with bounding boxes" << '\n';
-  Boxes<DeviceType> boxes(execution_space);
-  std::cout << "Bounding boxes set up." << '\n';
-
-  std::cout << "Creating BVH tree." << '\n';
-  ArborX::BVH<MemorySpace> const tree(execution_space, boxes);
-  std::cout << "BVH tree set up." << '\n';
-
-  std::cout << "Starting the queries." << '\n';
   // The query will resize indices and offsets accordingly
   Kokkos::View<int *, MemorySpace> indices("Example::indices", 0);
   Kokkos::View<int *, MemorySpace> offsets("Example::offsets", 0);
+  tree.query(space, queries, indices, offsets);
 
-  ArborX::query(tree, execution_space, boxes, indices, offsets);
-  std::cout << "Queries done." << '\n';
-
-  std::cout << "Starting checking results." << '\n';
   auto offsets_host =
       Kokkos::create_mirror_view_and_copy(Kokkos::HostSpace{}, offsets);
   auto indices_host =
       Kokkos::create_mirror_view_and_copy(Kokkos::HostSpace{}, indices);
 
-  unsigned int const n = boxes.size();
-  if (offsets_host.size() != n + 1)
-    Kokkos::abort("Wrong dimensions for the offsets View!\n");
-  for (int i = 0; i < static_cast<int>(n + 1); ++i)
-    if (offsets_host(i) != i)
-      Kokkos::abort("Wrong entry in the offsets View!\n");
-
-  if (indices_host.size() != n)
-    Kokkos::abort("Wrong dimensions for the indices View!\n");
-  for (int i = 0; i < static_cast<int>(n); ++i)
-    if (indices_host(i) != i)
-      Kokkos::abort("Wrong entry in the indices View!\n");
-  std::cout << "Checking results successful." << '\n';
+  // Expected output:
+  //   offsets: 0 1 2 6
+  //   indices: 3 3 0 2 1 3
+  // The order of the last 4 indices may vary.
+  std::cout << "offsets: ";
+  std::copy(offsets_host.data(), offsets_host.data() + offsets.size(),
+            std::ostream_iterator<int>(std::cout, " "));
+  std::cout << "\nindices: ";
+  std::copy(indices_host.data(), indices_host.data() + indices.size(),
+            std::ostream_iterator<int>(std::cout, " "));
+  std::cout << "\n";
```

Comment:
```suggestion
  std::cout << '\n';
```

Reply:
Not a bad idea, but not worth running CI again.